### PR TITLE
Bug #200404 - AG edit -> AG enrollment -> Photo receipt coming outside the box - It should come inside the box

### PR DESCRIPTION
**Pull request by Ekta29yadav**

### CODE REVIEW

File: apps/front-end/src/pages/front-end/BenificiaryListView.js [hunk 1]
```diff
@@ -35,13 +35,25 @@ const List = ({ data }) => {
               space="2"
             >
               <HStack justifyContent="space-between">
-                <HStack alignItems="Center" justifyContent="space-between">
+                <HStack
+                  alignItems="Center"
+                  justifyContent="space-between"
+                  flex="0.6"
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/BenificiaryListView.js:41]: use `flex="6"`

File: apps/front-end/src/pages/front-end/BenificiaryListView.js [hunk 2]
```diff
@@ -50,7 +62,7 @@ const List = ({ data }) => {
                     </FrontEndTypo.H5>
                   </VStack>
                 </HStack>
-                <Box>
+                <Box flex="0.3">
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/BenificiaryListView.js:65]: use `flex="3"`

File: apps/front-end/src/pages/front-end/ag-edit/choose-subjects/subjectForm.js [hunk 3]
```diff
@@ -608,23 +608,45 @@ export default function App({ facilitator, id, ip, onClick }) {
...
+                          width="full"
+                          height="172px"
+                          borderRadius="5px"
+                          borderWidth="1px"
+                          borderColor="worksheetBoxText.100"
+                          alignSelf="Center"
+                        />
+                      </VStack>
+                    </VStack>
+                    <Button
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/ag-edit/choose-subjects/subjectForm.js:638]: Use component button if needed.

**Review by sagarkoshti1990 — CHANGES_REQUESTED** [review]

File: apps/front-end/src/pages/front-end/BenificiaryListView.js (hunk 1 above)
Comment (Ekta29yadav) [reply apps/front-end/src/pages/front-end/BenificiaryListView.js]: done

File: apps/front-end/src/pages/front-end/ag-edit/choose-subjects/subjectForm.js (hunk 3 above)
Comment (Ekta29yadav) [reply apps/front-end/src/pages/front-end/ag-edit/choose-subjects/subjectForm.js:638]: done

File: apps/front-end/src/pages/front-end/BenificiaryListView.js (hunk 2 above)
Comment (Ekta29yadav) [reply apps/front-end/src/pages/front-end/BenificiaryListView.js]: done

**Pull request merged**